internal/environment/lab/network/dhcp/dhcp.go

```diff
@@ -69,40 +69,40 @@ func New(format func(n int) string) (*Server, error) {
 	})
 
 	return &Server{
-		cont:     cont,
-		confFile: confFile,
-		dns:      dns,
-		subnet:   subnet,
+		Cont:     cont,
+		ConfFile: confFile,
+		Dns:      dns,
+		Subnet:   subnet,
 	}, nil
```

Comment:
Why export everything?

Reply:
Same reason for iptables, needed when saving the state